src/fetch.js

```diff
@@ -24,10 +24,13 @@ const fetchStrapiContentTypes = async (pluginOptions) => {
   };
 };
 
-const fetchEntity = async ({ endpoint, queryParams, uid }, ctx) => {
+const fetchEntity = async ({ endpoint, queryParams, uid, pluginOptions }, ctx) => {
   const { strapiConfig, reporter } = ctx;
   const axiosInstance = createInstance(strapiConfig);
 
+  // Ignore queryParams locale in favor of pluginOptions
+  delete queryParams.locale;
```

Comment:
That might be a bit harsh if the pluginOptions are not set.
WDYT about only deleting the locale params when the `pluginOptions.i18n.locale` is defined? So the user can still keep the behaviour and use the source plugin even when i18n is not installed in a strapi app and a content type has field called `locale` ?